build.gradle

```diff
@@ -335,6 +335,9 @@ subprojects {
     maxParallelForks = Integer.valueOf(System.getProperty('maxParallelForks', '4'))
     minHeapSize = System.getProperty('minHeapSize', '1g')
     maxHeapSize = System.getProperty('maxHeapSize', '4g')
+
+    systemProperty 'pubsubBrokerFactory', System.getProperty('pubsubBrokerFactory', "com.linkedin.venice.integration.utils.KafkaBrokerFactory")
```

Comment:
Nit: do we need to change the `systemProperty` to 'pubSubBrokerFactory'?

Reply:
Done